extension/lib/bigcommerce/CustomerRepository.js

```diff
@@ -1,8 +1,10 @@
 const jwtDecoder = require('jwt-simple')
+const BigCommerce = require('node-bigcommerce')
+
 const BigCommerceCustomerTokenInvalidError = require('./customer/jwt/TokenInvalidError')
 const BigCommerceCustomerTokenUnverifiedError = require('./customer/jwt/TokenInvalidError')
```

Comment:
(I know this is not part of you change, but) it seems like the wrong error is being imported here
```suggestion
const BigCommerceCustomerTokenUnverifiedError = require('./customer/jwt/TokenUnverifiedError')
```